Solutions/Endjin.SemVer.DotNetApi.Cli/Endjin.SemVer.DotNetApi.Cli.csproj

```diff
@@ -1,42 +1,47 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
-  
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <NoWarn>RCS1090; SA1600; CS1591; CS1587</NoWarn>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <Authors>endjin</Authors>
-    <Product>Endjin.SemVer.DotNetApi</Product>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <PackageId>nupkgversion</PackageId>
-    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <PackageDescription>nupkgversion is a .NET Global Tools for comparing two NuGet packages and determining whether differences trigger SemVer Major, Minor or Patch changes</PackageDescription>
-    <PackageTags>nupkgversion pr-autoflow devops</PackageTags>
-    <PackageReleaseNotes></PackageReleaseNotes>
-    <PackAsTool>true</PackAsTool>
-    <ToolCommandName>nupkgversion</ToolCommandName>
-    <AssemblyName>nupkgversion</AssemblyName>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Endjin.RecommendedPractices" Version="1.1.0">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.7" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.7" />
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="3.1.7" />
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20371.2" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\Endjin.SemVer.DotNetApi\Endjin.SemVer.DotNetApi.csproj" />
-  </ItemGroup>
+	<Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
```

Comment:
You appear to have replaced all the spaces with tabs here. I'm not sure if that was deliberate.

If it was deliberate, you probably also need to do the thing to tell GitHub that in this context it should presume 4-space tab stops, because as it is, it has defaulted to 8, meaning the file looks different in GitHub from in VS. (I forget how to do that, but I thought I'd check whether you really meant to do a "replace spaces with tabs" as part of this PR before going and looking it up.)

If it was deliberate, you've only done it in this file, and nowhere else.